ddtrace/tracer.py

```diff
@@ -385,6 +386,10 @@ def start_span(self, name, child_of=None, service=None, resource=None, span_type
             span.set_tags(self.tags)
         if not span._parent:
             span.set_tag(system.PID, getpid())
+        # Set `version` tag based on `DD_VERSION` or configured `config.version` setting
+        # TODO: Only set this if `service` == `config.service` (`DD_SERVICE`)
```

Comment:
Maybe a dumb question: Why would we want this? Wouldn't we want all spans to be tagged with the app version?

Reply:
not if they represent execution in another service, like `sql.query` spans.

we want to make sure the version only relates to the primary service for the app.